test/wrapper_test.rb

```diff
@@ -3204,4 +3188,72 @@ def test_empty_result_when_no_mission
 
     assert_equal 2, result[:routes].size
   end
+
+  def test_assert_inapplicable_for_vroom_if_vehicle_distance
+    problem = VRP.basic
+    problem[:vehicles].first[:distance] = 10
+
+    vrp = TestHelper.create(problem)
+    assert_includes OptimizerWrapper.config[:services][:vroom].inapplicable_solve?(vrp), :assert_no_distance_limitation
+    refute_includes OptimizerWrapper.config[:services][:ortools].inapplicable_solve?(vrp), :assert_no_distance_limitation
+  end
+
+  def test_assert_inapplicable_vroom_with_periodic_heuristic
+    problem = VRP.scheduling
+    problem[:services].first[:visits_number] = 2
+
+    assert_includes OptimizerWrapper.config[:services][:vroom].inapplicable_solve?(TestHelper.create(problem)), :assert_only_one_visit
+  end
+
+  def test_assert_applicable_for_vroom_if_initial_routes
```

Comment:
Vroom should not be inapplicable due to initial routes 
https://github.com/Mapotempo/optimizer-api/pull/59

Reply:
test name says "applicable" :confused: but yeah we can use vroom if everything else is "right" even if there is initial solutuion

Reply:
Mea culpa !